renku/cli/dataset.py

```diff
@@ -31,7 +31,8 @@
 and ``--description`` options when creating a dataset. You can also set
 creator of a dataset by passing a ``--creator`` option along with a string
 that defines creator's name, email, and an optional affiliation. Pass multiple
-``--creator`` flags to add a list of creators.
+``--creator`` flags to add a list of creators. To pass a list of keywords for
+a dataset use ``--keyword`` flag.
```

Comment:
This is getting a bit crowded, maybe we could use a table instead? https://www.tablesgenerator.com/text_tables to save time :)

Reply:
Good idea!